src/main/java/sg/ncl/MainController.java

```diff
@@ -68,6 +68,10 @@
 
     private final String permissionDeniedMessage = "Permission denied. If the error persists, please contact " + CONTACT_EMAIL;
 
+    // for user dashboard hashmap key values
+    private final String userDashboardTeams = "teams";
+    private final String userDashboardRunningExperiments = "runningExperiments";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1170)